src/test/java/com/gavinfenton/quizolation/service/QuizServiceTest.java

```diff
@@ -144,6 +148,29 @@ public void testUpdateQuizSetsIdBeforeSaveAndReturnsFromRepository() {
         assertEquals(quizExpected, quizActual);
     }
 
+    @Test
+    public void testAddTeamToQuizCallsAndReturnsFromRepo() throws Exception {
+        //Given
+        Quiz quizSaving = new Quiz();
+        Long quizIdSavingExpected = 32L;
+        Quiz quizExpected = new Quiz();
+        Team teamSaving = new Team();
+        Long teamIdSavingExpected = 1L;
+        Team teamExpected = new Team();
+        given(teamService.getTeam(teamIdSavingExpected)).willReturn(teamExpected);
+        given(quizRepository.findById(quizIdSavingExpected)).willReturn(Optional.of(quizSaving));
+        given(quizRepository.save(quizSaving)).willReturn(quizExpected);
+
+        //When
+        Quiz quizActual = quizService.addTeamToQuiz(quizIdSavingExpected, teamIdSavingExpected);
+
+        //Then
```

Comment:
I would add a check to make sure that team has been added to the Quiz returned from getQuiz()

Reply:
resolved